src/docs/automation-and-resiliency/request-quota-adjustment-for-openshift-project-set.md

```diff
@@ -138,15 +142,18 @@ The Platform Services team needs to know if your application is using the curren
 
 * Once all the information has been compiled from [**Step 4**](#step-4-gather-information-for-your-request) click on the `Submit edit request` button
 
-* A pop-up will appear to allow space to include all the information you have collected and gathered, this place only allows for text information if you need to submit photos or files please do so in the next step as it is necessary and part of the request process
+* A pop-up will appear to allow space to include all the information you have collected and gathered. This place only allows for text information
 
-### Step 6: Email the Platform Services team
+### Step 6: Wait for an email from the Platform Services team
 
-The last step is to email the request to the Platform Services team at [PlatformServicesTeam@gov.bc.ca](mailto:PlatformServicesTeam@gov.bc.ca) and include the following information: 
+For the last step, you'll receive an email from the Platform Services team providing you with approval or requesting further information from your quota increase request (remember that quota decreases are automatically approved). 
 
-* Attach screenshot of Sysdig dashboard (if available)
-* Attach additional images or files that supplement the already submitted information in the previous step
-* If you have further questions, it is also recommended to book a meeting with the Platform Services Team
+If the Platform Services Team requires more information, they'll likely ask for: 
+
+* A screenshot of Sysdig dashboard (if available)
+* Additional images or files that supplement the already submitted information in the previous step
```

Comment:
instead of file, maybe some description of what's the plan, what they are trying to do would make more sense